inference-engine/tests/ngraph_functions/src/input_layer.cpp

```diff
@@ -17,7 +17,7 @@ std::shared_ptr<ngraph::Node> makeInputLayer(const element::Type &type, ngraph::
     std::shared_ptr<ngraph::Node> input;
     switch (inputType) {
         case ngraph::helpers::InputLayerType::CONSTANT: {
-            std::vector<float> data(ngraph::shape_size(shape));
+            std::vector<float> data(ngraph::shape_size(shape), 3.f);
```

Comment:
Why we need this change?

Reply:
Because earlier this vector was initialized with float() values, which means zeroes. For example, if we check additions, we will add zeroes in this case that will not change initial data, and if the addition somehow doesn't work, we won't see the difference. So number 3 is a better solution because it will change numbers in all arithmetical operations.

Reply:
Wouldn't it better to fill data with random values inside makeConstant? Broadcasting the same value for whole tensor can hide some bugs as well.

Reply:
Changed in beeab5943a30713a7e27e9a2ccc4dca24fb99457. The data is now filled with random values.